README.md

```diff
@@ -26,12 +26,30 @@ $ sh build.sh
 
 ## Demo
 ### gauss_pdf_and_cdf.py
-Show and save {(gaussian, beta), (probability distribution function, cumulative distribution function)} figure.
 
-If you specify "gaussian" as the argument of the main function, show gaussian pdf and cdf,
+Show and save some distribution figures.
+
+The figures supported in this demo are shown at the follow table.
+|distribution type|function type|
+|:---|:---|
+|gaussian|PDF|
```

Comment:
Is this clear what PDF and CDF here? Most users see documents from the top to the bottom. Also, they see emphasized structures such as table first.